powersimdata/design/generation/clean_capacity_scaling.py

```diff
@@ -462,7 +464,7 @@ def create_change_table(input_targets, ref_scenario):
         prev_wind = input_targets.loc[region, "wind.prev_capacity"]
         next_solar = input_targets.loc[region, "solar.next_capacity"]
         next_wind = input_targets.loc[region, "wind.next_capacity"]
-        zone_names = area_to_loadzone(base_grid, region)
+        zone_names = area_to_loadzone(region)
```

Comment:
Does this now need to be `area_to_loadzone(base_grid.get_grid_model(), region)` instead of `area_to_loadzone(base_grid, region)` or `area_to_loadzone(region)`?

Reply:
We need to change things there (and in other modules such as in `scenario_info`). We need to instantiate a `ModelImmutables` class and use the `area_to_loadzone` method.

In fact, before we merged your branch everything was backward compatible. Now we have to be careful and also prepare a PR in PostREISE with this new design that we will merge in parallel than `model`

Reply:
I take care of updating the various modules

Reply:
And it is a good thing @danielolsen that your commits have been rebased because I will be able to sneak the changes in your commits

Reply:
I am very good at causing problems.

Reply:
You forsee changes that will have to happen! Refactoring PostREISE to abstract the grid was next. We just have to do it now.

Reply:
Done